spec/controllers/admin/books_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Admin::BooksController, :type => :controller do
+  render_views
+
+  let(:book) { Book.create!(id: 1, title: "You wouldn't") }
+  let(:admin_user) { AdminUser.create!(email: 'admin@example.com', password: 'password') }
+
+  before(:each) do
+    sign_in admin_user
+  end
+
+  describe "GET index" do
+    it 'returns http success' do
+      get :index
```

Comment:
This is all very cool, Deepika, but I don't see why you couldn't do this in a regular controller test — did that not work at all? Happy to talk more. Thanks!

Reply:
@thisisstephenbetts we already have regular controller test cases.  But it will not connect to active-admin .

see below image regular controller test cases already have  for book_controller.

<img width="1096" alt="Screen Shot 2021-01-26 at 9 42 14 AM" src="https://user-images.githubusercontent.com/30579702/105860052-2d644080-5fbb-11eb-9a10-03a6762c5e20.png">